guides/plugins/apps/custom-data/custom-entities.md

```diff
@@ -40,6 +40,43 @@ Additionally, to the repository you can also access your custom entities via [Ad
 POST /api/search/custom-entity-blog
 ```
 
+## Custom Fields
+
+By default, it is not possible to create a custom field of type "Entity Select" which references a custom entity. However, you can opt in to this behavior. You will need to add the `custom-fields-aware` & `label-property` attributes to your entity definition:
+
+{% code title="Resources/entities.xml" %}
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/System/CustomEntity/Xml/entity-1.0.xsd">
+    <entity name="custom_entity_bundle" custom-fields-aware="true" label-property="name">
+        <fields>
+            <string name="name" required="true" translatable="true" store-api-aware="true" />
+            <price name="discount" required="true" store-api-aware="true"/>
+            <many-to-many name="products" reference="product" store-api-aware="true" />
+        </fields>
+    </entity>
+</entities>
+```
+
+{% endcode %}
```

Comment:
The first part of this article has a code block with similar info to register the custom entities as below:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/System/CustomEntity/Xml/entity-1.0.xsd">
    <entity name="custom_entity_bundle">
        <fields>
            <string name="name" required="true" translatable="true" store-api-aware="true" />
            <price name="discount" required="true" store-api-aware="true"/>
            <many-to-many name="products" reference="product" store-api-aware="true" />
        </fields>
    </entity>
</entities>
```
May I know if the new content is a replacement for that?

Reply:
Almost, we add the attributes: `custom-fields-aware="true" label-property="name"` to the `<entity` line. Is there a better way to express the changes ?

Reply:
I think we can keep it as is, as shared by you because they are relevantly different topics. 👍